src/lib/y2users/linux/users_writer.rb

```diff
@@ -217,9 +225,24 @@ def clear_home(user, issues)
         # Pathname.new(user.home).children.each { |f| f.rm_rf }
       rescue Cheetah::ExecutionFailed => e
         issues << Y2Issues::Issue.new(
-          format(_("Something was wrong while cleaning up '%s'"), user.home)
+          format(_("Cannot clean up '%s'"), user.home)
+        )
+        log.error("Error cleaning up '#{user.home}' - #{e.message}")
+      end
+
+      def chown_home(user, issues)
+        return unless user.home&.path
+        return unless File.exist?(user.home.path)
+
+        owner = user.uid
+        owner << ":#{user.gid}" if user.gid
+
+        Yast::Execute.on_target!("/usr/bin/chown", "-R", owner, "\"#{user.home.path}\"")
```

Comment:
For consistency with rest of executed commands, I'd use a private constant for chown path.